### DISCUSSION TITLE
Browsing security & parental control web service are not available in China? Or it's blocked by my ISP?

### DISCUSSION THREAD

**Discussion by maidmeow4** [AdguardTeam/AdGuardHome#4491] [Q&A] [answered]
Enabled both services, then all DNS queries timed out, verbose log of AGH shows it was unable to connect to `dns-family.adguard.com`.
![photo_2022-04-14_20-01-41](https://user-images.githubusercontent.com/8447650/163393074-2f71dd7f-b526-4851-b680-ed6880a7837d.jpg)

Tried to investigate this issue with `curl`, it says `curl: (35) schannel: failed to receive handshake, SSL/TLS connection failed`, both IPv4 and IPv6.
![2022-04-14_204727](https://user-images.githubusercontent.com/8447650/163393582-1a46b9de-1617-478a-8a9e-6148e92bdfaa.png)

Disabling those services will bring back the DNS service, but I would like to use those services if possible.

**Comment by ainar-g** [comment, marked as answer]
It looks like it, and unfortunately that means that you won't be able to use those features unless you use some kind of a proxy to bypass the block.

**Reply by maidmeow4** [reply]
Is this API hardcoded or I can change it somewhere in the config file?

**Reply by ainar-g** [reply]
@maidmeow4, yes, in `internal/filtering/safebrowsing.go`.  See [this article][1] for more info on how the feature works.  (This article is about other AdGuard products, but AGH works basically the same way.)

[1]: https://kb.adguard.com/en/general/how-malware-protection-works#adguard-for-windows-mac-and-android
